### ISSUE TITLE
no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no nono no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no no

### ISSUE THREAD

**Issue by ghost** [Chocohead/Rift#48]
this is stupid forge is already good this is bad fuck you delete this shit ur raft sucks use forge mother fuckers rift sucks forge is perfect awesome good for mod needs no raft is hacks hacks HACKS 113 stupid kill your self

**Comment by hYdos** [comment]
no :)

**Comment by Ueaj-Kerman** [comment]
rift is crob

**Comment by 22858** [comment]
yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes yes 
forge is bad
